reactive/cwr.py

```diff
@@ -107,6 +107,12 @@ def install_jenkins_jobs(connected_jenkins):
         if not installed:
             hookenv.log("installation of {} did not complete on time."
                         .format(plugin))
+            # Retrying does not play well here.
+            # I have seen a case where Jenkins mistakenly reports that
+            # the plugin is installed causing an infinite retry loop :(
+            # Here we report that jenkins failed and we block.
+            set_state("jenkins.jobs.failed")
```

Comment:
Ack @ktsakalozos, i understand the issue better now.  Jenkins may never be able to install its plugins (e.g. if behind a proxy), so re-adding the cwr relation will simply block again.  That's fine for this PR as we want to fail as fast as possible to let the user know `cwr` won't work.

Thanks for the explanation.